packages/create-app/src/commands/init.ts

```diff
@@ -37,9 +37,9 @@ export default class Init extends Command {
       env: 'SHOPIFY_FLAG_TEMPLATE',
     }),
     // eslint-disable-next-line @typescript-eslint/naming-convention
-    'dependency-manager': Flags.string({
+    'package-manager': Flags.string({
```

Comment:
Did we have this flag documented somewhere?

Reply:
This is a hidden flag that's only used by acceptance tests to test workflows with different package managers.